v2/robotmk/tests/timeout/lib/create_child.py

```diff
@@ -0,0 +1,6 @@
+import subprocess
+
+
+def spawn() -> None:
+    with subprocess.Popen(["python", "-c", "import time; time.sleep(10)"]) as p:
```

Comment:
We can directly use `time.sleep` without a subprocess

Reply:
Done